dev/reference/performance/statement-summary.md

```diff
@@ -0,0 +1,148 @@
++-------------+------------+------------------------------------------+
+|     1042040 |          2 | select * from employee where name='eric' |
+|      345053 |          3 | select * from employee where id=3100     |
++-------------+------------+------------------------------------------+
+2 rows in set (0.00 sec)
+```
+
+### Which categories of SQL statements consume the longest total time?
+
+To tune the system, we can find out the top 3 categories of SQL statements with the longest time consumption:
+
+```sql
+SELECT sum_latency, avg_latency, exec_count, query_sample_text
+    FROM performance_schema.events_statements_summary_by_digest
+    ORDER BY sum_latency DESC LIMIT 3;
+```
+
+The result shows that the following three SQL categories consume the longest time in total, which require focus in optimization.
+
+```
++-------------+-------------+------------+-----------------------------------------------------------------------+
+| sum_latency | avg_latency | exec_count | query_sample_text                                                     |
++-------------+-------------+------------+-----------------------------------------------------------------------+
+|     7855660 |     1122237 |          7 | select avg(salary) from employee where company_id=2013                |
+|     7241960 |     1448392 |          5 | select * from employee join company on employee.company_id=company.id |
+|     2084081 |     1042040 |          2 | select * from employee where name='eric'                              |
++-------------+-------------+------------+-----------------------------------------------------------------------+
+3 rows in set (0.00 sec)
+```
+
+## Configurations
+
+The statement summary feature is disabled by default. You can enable it by setting a system variable, for example:
+
+```sql
+set global tidb_enable_stmt_summary = true;
+```
+
+The `tidb_enable_stmt_summary` system variable has two scopes - global and session, which work a little differently from other system variables, as described below:
+
+- Set the global variable to apply to the cluster immediately.
+- Set the session variable to apply to the current TiDB server immediately. This is useful when you debug on a single TiDB server instance.
+- The session variable has a higher read priority. The global variable  will be read only if no session variable is set.
+- Set the session variable to a blank string to re-read the global variable.
+
+The statistics in the system table will be cleared if the statement summary feature is disabled, and will be re-calculated next time the statement summary feature is enabled. Tests have shown that enabling this feature has little impact on performance.
+
+`events_statements_summary_by_digest` is a memory table. To prevent potential memory issues, we need to limit the number of statements to be saved and the longest SQL display length. You can configure these limits using the following parameters under [stmt-summary] of config.toml:
```

Comment:
```suggestion
`events_statements_summary_by_digest` is a memory table. To prevent potential memory issues, we need to limit the number of statements to be saved and the longest SQL display length. You can configure these limits using the following parameters under `[stmt-summary]` of `config.toml`:
```